diff --git a/Makefile.target b/Makefile.target
index 5d5d19f..ebccee2 100644
--- a/Makefile.target
+++ b/Makefile.target
@@ -160,11 +160,11 @@ $(foreach lib,$(LIBS),$(eval $(call LIB_template,$(lib))))
 
 ## Generated help messages
 ifneq ($(XSLTPROC),)
-%_help.h: $(SRCDIR)/man/%.xml $(SRCDIR)/man/header.xslt
+%_help.h: $(SRCDIR)/man/tools/%.xml $(SRCDIR)/man/tools/header.xslt
 	@echo "Generating $@ for $(TARGET)"  $(MSGLOG)
-	$(call CMD_template,$(XSLTPROC) $(SRCDIR)/man/header.xslt $< > $@,$(LOG))
+	$(call CMD_template,$(XSLTPROC) $(SRCDIR)/man/tools/header.xslt $< > $@,$(LOG))
 else
-%_help.h: $(SRCDIR)/man/%.xml $(SRCDIR)/man/header.xslt
+%_help.h: $(SRCDIR)/man/tools/%.xml $(SRCDIR)/man/tools/header.xslt
 	@echo "Generating dummy $@ for $(TARGET)"  $(MSGLOG)
 	@echo $(MSGLOG)
 	@echo '/* This file was generated, do not edit. */' > $@
diff --git a/configure b/configure
index 41e710a..3027852 100755
--- a/configure
+++ b/configure
@@ -373,7 +373,7 @@ fi
 ##
 
 case "$cpu" in 
-    i[3-9]86|x86_64|alpha)
+    i[3-9]86|x86_64|alpha|arm64|aarch64)
 	endian=little
 	;;
     ppc|powerpc|sparc)
